usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -716,8 +716,7 @@
                     "primary_place_of_performance_foreign_location",
                     "award__latest_transaction__assistance_data__place_of_performance_forei",
                 ),
-                ("cfda_number", "award__latest_transaction__assistance_data__cfda_number"),
-                ("cfda_title", "award__latest_transaction__assistance_data__cfda_title"),
+                ("cfda_numbers_and_titles", None),  # Annotation is used to create this column
```

Comment:
Did requirements get clarified on the story to _combine_ what was 2 columns into 1? Originally I think Ross had it written out as keeping 2 cols, and just ordering the numbers and titles in each column the same so they "lined up".

Reply:
There wasn't really any clarification on which method was best, I personally think it works better UX-wise and with the fact that we have DEFC, program activity, and object class the same way having the titles and numbers together, but it's trivial to separate them so I'll change it if that makes more sense to people.